pkg/templates/translator/translator.go

```diff
@@ -18,13 +18,13 @@ type Translator interface {
 }
 
 type Options struct {
-	Name        string
-	Default     any
-	Description string
-	Sensitive   bool
-	Order       int
-	TypeExpress any
-	Nullable    bool
+	Name          string
+	DefaultValue  any
+	DefaultObject any
```

Comment:
Why do we need both fields?

Reply:
DefaultObject is used for passing the default for nested object.